### PR TITLE
Add HireFire to account list

### PR DESCRIPTION
**Pull request by codingjoe**

### CODE REVIEW

File: accounts.md [hunk 1]
```diff
@@ -16,6 +16,12 @@ Make sure to add your name and any picture (does need to be you) to help your ne
 
 We use Heroku Teams, please create an account on Heroku and ask your team lead to give you the necessary permissions.
 
+#### HireFire
+
+HireFire is used incombination with Heroku for load based auto-scaling.
+Please setup an account under voiio E-Mail address and enable 2FA.
+Only after completing both step can you be added to our HireFire organization.
```
Comment (fidelitas) [inline accounts.md:23]:
```suggestion
We use [HireFire][hirefire] in combination with Heroku for load based auto-scaling.
Please, setup an account with your voiio E-Mail address and enable "2FA".
Only after completing both steps, we can add you to our "HireFire"-organization. Please refer to your team lead for adding you to the organization.

[hirefire]: https://hirefire.io/
```

**Review by fidelitas — CHANGES_REQUESTED** [review]
Proposed some improvements.

**Comment by codingjoe** [comment]
Thx @fidelitas 
FYI, we usually reserve `request changes` reviews, if we want to halt deployment or veto a change. For anything else, we use comments or approve with comments for optional improvements. GitHub has done a poor job with the UI here, and change requests can come on strong or discouraging. Comments have yielded more constructive discussions for us in the past.

**Comment by fidelitas** [comment]
@codingjoe thanks for letting me know.
Do you want me to do anything for this PR for now?

**Comment by fidelitas** [comment]
Oh. It's merged. Never mind.

**Comment by codingjoe** [comment]
No, but you should get a hirefire account and added to our org, so you can disable or change it, if need be.

**Pull request merged**